src/index.html

```diff
@@ -16,17 +17,34 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion-->
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="form-search form-search--big"
+    >
+      <label class="form-search__label form-search__label--big">
+        <input
+          class="form-search__input form-search__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try &QUOT;Los Angeles&QUOT;"
```

Comment:
The placeholder attribute contains `&QUOT;`, which is incorrect. Please replace it with the correct HTML entity `&quot;` for proper encoding.